examples/helm-charts/zarf.yaml

```diff
@@ -16,6 +16,13 @@ components:
         localPath: chart
         valuesFiles:
           - values.yaml
+        # Variables are used to override the default values in the chart
+        # This can be overridden by the user at deployment time with the `--set` flag
+        variables:
+          - name: REPLICA_COUNT
+            description: "Override the number of pod replicas"
+            path: replicaCount
+            value: 2
```

Comment:
`value: 2` is not being respected here

you'll need to add it as a field in the `ZarfChartVariable` struct and update the unit tests in `deploy_test.go` to verify the expected priorities/behavior is correct

Reply:
although I will note that @Racer159 didn't mention a `value:` field in the issue:

https://github.com/defenseunicorns/zarf/issues/2133#issuecomment-1981471500

so may be worth more discussion whether that is wanted/needed

Reply:
@lucasrod16 Trying to add some context here 😃:

 In uds-cli the [field](https://github.com/defenseunicorns/uds-cli/blob/54499a70e7dda5082fb424ba305dc19da0b0ab5d/src/types/bundle.go#L54) under variables corresponding to `values:` here is `default:`. 

From a bundle perspective, `value` and `variable` overrides are technically two different things (create time static vs deploy time override). 

I believe the intent behind the issue was to expose the same `variable:` syntax from bundles into zarf and it just happened that the referenced examples @Racer159 provided only had variables without a `default:`. [Here](https://github.com/defenseunicorns/uds-software-factory/blob/bfd10ff3c519583989804d5d28e71b17e76f9628/bundles/dev/uds-bundle.yaml#L78) is an example that uses the `default:` field. 

Definitely defer to @Racer159  as to whether or not that was omitted intentionally.

Reply:
Sorry yeah I missed this in my review - value shouldn't be there - description mirth also be droppable too actually - for defaults my though was you could define it (along with prompts and other variable things) in the higher-level `variables` field at the package level

Reply:
@zachariahmiller thanks for the context/clarification! I think it makes sense to stay aligned with uds-cli and include a `default:` field here. I don't have a strong preference either way since this feature is primarily for `swf`/`uds`, so whatever is the most helpful for that use case is okay with me.

Reply:
I have removed the `value`. Thanks